### ISSUE TITLE
Case insensitive input for focus sub-command

### ISSUE THREAD

**Issue by eyalev** [smblott-github/chromix-too#10]
Is there a way to treat the input to the `focus` sub-command as case insensitive?

Thanks

**Comment by smblott-github** [comment]
No; not currently.  The filtering is done [here](https://github.com/smblott-github/chromix-too/blob/master/client.coffee#L46), if you're interested in adding something.
